specs/core/0_beacon-chain.md

```diff
@@ -507,16 +490,12 @@ class BeaconState(Container):
     validators: List[Validator, VALIDATOR_REGISTRY_LIMIT]
     balances: List[Gwei, VALIDATOR_REGISTRY_LIMIT]
     # Shuffling
```

Comment:
```suggestion
    # Randomness
```
Agree with Justin here. I tried keeping randomness separate from shuffling in ZRNT, and it was not pretty. This is great :)